src/main/java/com/google/songgame/servlets/ChatServlet.java

```diff
@@ -119,16 +128,16 @@ private String getUsername(String userId) {
     return (String) currentUser.getProperty("username");
   }
 
-  // TODO: @salilnadkarni update checkStatus / updateStatus to use Datastore
-  private boolean checkStatus(String userId) {
-    if (!status.containsKey(userId)) {
-      status.put(userId, false);
-    }
-    return status.get(userId) == true;
+  private boolean checkGuessStatus(String userId, Entity currentRound) {
+    EmbeddedEntity userGuessStatuses = (EmbeddedEntity) currentRound.getProperty("userGuessStatuses");
+    boolean userGuessStatus = (Boolean) userGuessStatuses.getProperty(userId);
+    return userGuessStatus == true;
```

Comment:
:)

think about this

bool userGuessStatus.

so its actually true | false

so 134 is doing

return true == true or return false == true.

in which case, you can just omit the ` == true` and just return `(Boolean) userGuessStatuses.getProperty(userId);`